### DISCUSSION TITLE
Redock a window

### DISCUSSION THREAD

**Discussion by noah-severyn** [rubberduck-vba/Rubberduck#6131] [Q&A] [answered]
I inadvertently unchecked the "dockable" checkbox on the 'Find all references' window. How do I redock that window (or any other Rubberduck window for that matter)?

**Comment by noah-severyn** [comment, marked as answer]
On further research, I found the answer in [this comment](https://github.com/rubberduck-vba/Rubberduck/issues/3214#issuecomment-440673949) in #3214 , but is there a more "friendly" way to do this?

**Reply by retailcoder** [reply]
Aye, VBIDE's docking system is an absolute mess! Per PR #5203, there should be a context menu that makes it easier: https://github.com/rubberduck-vba/Rubberduck/pull/5203

**Reply by noah-severyn** [reply]
Strange... my problem is I do not get that dockable context window when a window is *not* docked. I am able to uncheck the checkbox to undock the window, but cannot get the context window to appear to recheck the box to dock it.

Oh well, at least there's a fix.
